### ISSUE TITLE
Menu sidebar not reachable via tab key

### ISSUE THREAD

**Issue by stefanreifenberg** [bigomics/omicsplayground#136]

**Comment by JohnCoene** [comment]
What do we want the tab key to do exactly here?

**Comment by stefanreifenberg** [comment]
In an ideal world the app should be fully usable via keyboard. That means all usable elements should be accessible using tab, especially dropdowns and filters. That means setting a tabindex where it is needed and working with focus areas. 
This is the first step towards an accessible app.

**Comment by stefanreifenberg** [comment]
Found this amazing chrome addon showing you the tab hierarchy on any webpage: https://chrome.google.com/webstore/detail/taba11y/aocppmckdocdjkphmofnklcjhdidgmga?hl=en
![Screenshot 2022-05-02 at 12 24 47](https://user-images.githubusercontent.com/15939539/166220395-927ee5f2-5fd7-4faf-98c5-2cd5caabb9ec.png)
